tico/experimental/quantization/ptq/wrappers/quant_elementwise.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2025 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Callable, Optional
+
+import torch
+import torch.nn as nn
+
+from tico.experimental.quantization.ptq.quant_config import QuantConfig
+from tico.experimental.quantization.ptq.wrappers.quant_module_base import (
+    QuantModuleBase,
+)
+from tico.experimental.quantization.ptq.wrappers.registry import register
+
+
+class QuantElementwise(QuantModuleBase):
+    """
+    Generic wrapper for *any* 1-to-1 element-wise op `y = f(x)`.
+
+    Sub-classes only need to implement:
+        • `FUNC`: a Callable that maps tensor→tensor
+    """
+
+    # subclass must set this
+    FUNC: Callable[[torch.Tensor], torch.Tensor] = staticmethod(lambda x: x)
```

Comment:
If it's must be set, why not "throw Exception"?

Reply:
I've added `__init_subclass__` for this. Thanks.